ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="trips-modal-container">
+  <app-pre-loading
+    [isLoading]="loading"
+    [size]="loaderSize"
+  ></app-pre-loading>
+  <header class = "form-row">
+    <h1 mat-dialog-title class = "trips-title">History</h1>
+    <i class="material-icons">
+      history
+    </i>
+  </header>
+  <div mat-dialog-content class="trips"
+       >
+    <div
+      class="trips-user"
+      *ngFor="let trip of tripsArray; trackBy: trackByFn"
```

Comment:
pls use limit to pipe to limit number of visible trips

Reply:
used limit